input/fsh/omrs-related-person.fsh

```diff
@@ -0,0 +1,137 @@
+Profile: OMRSRelatedPerson
+Parent: RelatedPerson
+Id: omrs-related-person
+Title: "OpenMRS Related Person"
+Description: "A FHIR RelatedPerson as understood by OpenMRS Relationship"
+* id 1..1
+* patient 1..1
+* identifier 1..1
+* address 0..*
+* name 0..*
+* gender 0..1
+* birthDate 0..1
+* active 1..1
+* period 1..1
+
+//OpenMRS RelatedPerson Example
+
+Instance: RelatedPersonExample
+InstanceOf: OMRSRelatedPerson
+Title: "Example of RelatedPerson"
+Description: "An example instance of the OpenMRS RelatedPerson resource."
+* patient = Reference(example-openmrs-Patient)
+* identifier.system = "http://openmrs.org/fhir/RelatedPerson"
+* identifier.value = "Person/1000Y"
+* name.id = "915c9146-8bdb-4fad-bc51-4af10d041c86"
+* name.family = "John"
+* name.given[0] = "Smith"
+* gender = #male
+* birthDate = "2001-09-08"
+* active = true
+* period.start = "2020-01-01"
+* period.end = "2024-01-01"
+
+Mapping: RelatedPersonMapping
+Source: OMRSRelatedPerson
+Target: "Relationship"
+Id: relatedPerson-mapping
+Title: "FHIR HL7 vs OMRS"
+Description: "This mapping defines how a FHIR RelatedPerson resource corresponds to an OpenMRS Relationship. In OpenMRS, a Relationship connects two Person objects, while in FHIR, a RelatedPerson represents a person with a connection to a patient."
+* -> "OMRS Relationship" "This profile maps to RelatedPerson in OMRS FHIR2"
+* identifier -> "Relationship.personA.uuid"
+* identifier.system -> "http://openmrs.org/fhir/RelatedPerson"
+* identifier.value -> "Person/{personA.uuid}"
+* active -> "voided"
+* name.family -> "Relationship.personA.names.family"
+* name.given -> "Relationship.personA.names.given"
+* gender -> "Relationship.personA.gender"
+* birthDate -> "Relationship.personA.birthdate"
+* patient -> "Relationship.personB"
+* period.start -> "Relationship.startDate"
+* period.end -> "Relationship.endDate"
+
+// Search parameters for OMRSRelatedPerson
```

Comment:
Extend the search parameters, The search parameters at the bottom of the [page ](https://build.fhir.org/relatedperson.html)should be able to guide you

Reply:
Updated to add missing search params